### PR TITLE
[wg/immersive-web] Changed text around LS

### PR DESCRIPTION
**Pull request by himorin**
/cc @Yonet @AdaRoseCannon @cwilso 

As email, alternate text per comment.

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
That looks good to me, it says yearly is there a particular time each year we should publish?

**Comment by himorin** [comment]
> That looks good to me, it says yearly is there a particular time each year we should publish?

I don't think so - both it is defined by the WG and even CRS publication is stated as `where Working Group will publish new Candidate Recommendations in timely after substantive changes` without any specific point (like month) in year or chartered period.
If WG want to schedule publication as fixed point basis, I believe that should be fine since nothing is defined in charter.

File: 2024/immersive-web-wg.html [hunk 1]
```diff
@@ -252,8 +252,10 @@ <h3>
           Normative Specifications
         </h3>
         <p>
-          The Working Group will deliver the following W3C normative specifications and WebXR Living Standards as noted
-          under the module deliverables section:
+          The Working Group will deliver the following W3C normative specifications: <br>
+          Some specifications have their Expected W3C Recommendation as yearly basis, and 
+          Working Group will continue to maintain specifications to support new XR hardware as a living specification 
+          where Working Group will publish new Candidate Recommendations in timely after substantive changes.
```
Comment (chrisn) [inline 2024/immersive-web-wg.html]:
```suggestion
          Some specifications are expected to be published as W3C Recommendations on a yearly basis, and 
          the Working Group will continue to maintain specifications to support new XR hardware as living specifications 
          where the Working Group will publish new Candidate Recommendations after substantive changes. 
```

Comment (chrisn) [reply 2024/immersive-web-wg.html]: This is just a wording suggestion, which I hope matches the WG's intent. This paragraph doesn't need updating from my point of view, as the changes below to indicate the "Draft state" as CR all I was suggesting really.

Comment (himorin) [reply 2024/immersive-web-wg.html]: @chrisn Really thank you for correcting English. I believe we are happy with this.

@Yonet @AdaRoseCannon @cwilso  ok to go with this? If ok, I'll go to all responders.

Comment (AdaRoseCannon) [reply 2024/immersive-web-wg.html]: Approved

**Comment by himorin** [comment]
merging. no comment provided during [1 week period](https://lists.w3.org/Archives/Member/member-charters-review/2024Sep/0005.html).

**Pull request merged**